tb_db/parsers.py

```diff
@@ -292,26 +290,39 @@ def parse_complex(complex_path):
     return my_dict
 
 def parse_species(speciation_path):
-    species = []
+    species = {}
     
     with open(speciation_path, 'r') as f:
         reader = csv.DictReader(f)
-        count = 0
+        #count = 0
```

Comment:
Let's remove this `count` variable if it isn't going to be used rather than leaving it commented out